### PR TITLE
Mobile menu toggle Pull Requst

### PR DESCRIPTION
**Pull request by iammouaz**
Hi There,

Kindly I need your help to check my code 💯
that's list for what I did on this pull request 🔢

1. - Add new division for mobile menu  
2. - add toggle function to the hamburger menu to show style
3. - Modification on style file to fix all problems 
4. - fixing all linters errors 

Thank you in advance 👏👏

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -30,18 +32,44 @@
             <a class=menu_item>Contact</a>
             <img class="menu_mail" src="./img/mail.png" alt="Email Icon">
         </nav>
-        <nav class="nav-toggle" id="navToggle">
+        <nav id="navToggle" class="nav-toggle">
             <img src="./img/menu.png" alt="Hamburger Menu">
+
+            <div class="Hamburger-me">
+                <img class="close" id="cancel" src="./img/Cancel.png" alt="Close">
```
Comment (rloterh) [inline index.html:39]: - [ ] Your _close button_ is **overlapping** with the menu items. Kindly improve upon this aspect by ensuring that the _close button_ is positioned at the same place as the _hamburger menu_, and not cluttered with the mobile menu items

**Your mobile menu overlay (when the hamburger menu is clicked):**
image

File: index.html [hunk 2]
```diff
@@ -20,6 +20,8 @@
 
 <body class="body">
 
+
+
```
Comment (rloterh) [inline index.html:24]: - [ ] OPTIONAL
Kindly ensure not to add leave empty lines in your code which are not serving any purpose, so as to keep your code cleaner

**Review by rloterh — CHANGES_REQUESTED** [review]
Hi Moaz, 

You did extremely great on this project! 👏 👏 
There are however a few issues that you still need to work on to go to the next project but you are almost there! 🚦  🚦  

## Good Points 👍  
- No linter errors


## Required Changes ♻️
_Kindly check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

_Kindly check the comments marked [OPTIONAL] under the review._

<br>
Cheers and Happy coding!👏 💻 🍷

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have 2 more limited reviews per this project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 3]
```diff
@@ -30,18 +28,27 @@
             <a class=menu_item>Contact</a>
             <img class="menu_mail" src="./img/mail.png" alt="Email Icon">
         </nav>
-        <nav class="nav-toggle" id="navToggle">
-            <img src="./img/menu.png" alt="Hamburger Menu">
+        <nav id="navToggle" class="nav-toggle">
```
Comment (carlos-ssh) [inline index.html:31]: - Your hamburger button isn't displayed, kindly fix that:
Screen Shot 2021-09-07 at 6 03 31

**Review by carlos-ssh — CHANGES_REQUESTED** [review]
Hi @iammouaz , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 3 above)
Comment (iammouaz) [reply index.html:31]: can you please check again because its fixed now

Comment (iammouaz) [reply index.html:31]: @carlos-ssh

**Review by george-shammar — APPROVED** [review]
Hi @iammouaz , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 

:heavy_check_mark: Your hamburger icon is now visible and works as expected. :+1: 

Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**